events/_event-post-macros.html

```diff
@@ -37,40 +38,38 @@
 
    post: A post from a query result.
 
-   options (optional): An object to use for customizing a event location image using
-                       Google Maps.
-
-   options.class:    A class name to be applied to the img element.
+   options (optional):         An object to use for customizing a event location image using
+                               Google Maps.
 
-   options.location: A location as either a comma-separated {latitude,longitude}
-                     pair (e.g. "40.714728,-73.998672") or a string address
-                     (e.g. "city hall, new york, ny").
+   options.additional_classes: Extra classes you wish to add to the image, space separated;
 
-   options.scale:    Affects the number of pixels that are returned.
-                     Accepted values are 1, 2 and 4.
+   options.locations:          A location as either a comma-separated {latitude,longitude}
+                               pair (e.g. "40.714728,-73.998672") or a string address
+                               (e.g. "city hall, new york, ny").
 
-   options.size:     Defines the rectangular dimensions of the map image.
-                     {horizontal_value}x{vertical_value} (e.g. 500x400)
-
-   options.zoom:     Determines the magnification level of the map.
+   options.scale:              Affects the number of pixels that are returned.
+                               Accepted values are 1, 2 and 4.
 
+   options.size:               Defines the rectangular dimensions of the map image.
+                               {horizontal_value}x{vertical_value} (e.g. 500x400)
 
+   options.zoom:               Sets pixel density for high-dpi devices.
 
    ========================================================================== #}
 
 {% macro event_location_image(post, options) %}
-
-    {%- set location = (post.location or 'Washington DC') -%}
-    {%- set scale = ('&scale=' + (options.scale or '2')) -%}
-    {%- set size = ('&size=' + (options.size or '276x155')) -%}
-    {%- set zoom = ('&zoom=' + (options.zoom or '12')) -%}
+    {%- set location = post.location | default('Washington DC', true) -%}
```

Comment:
@anselmbradford , I think we just need to encode the location as it might contain spaces.